app/src/pages/inside/dashboardItemPage/dashboardItemPage.jsx

```diff
@@ -46,33 +47,50 @@ const messages = defineMessages({
 });
 
 @injectIntl
-@connect((state) => ({ dashboardId: activeDashboardIdSelector(state) }))
+@connect((state) => ({
+  activeProject: activeProjectSelector(state),
+}))
 export class DashboardItemPage extends Component {
   static propTypes = {
     intl: intlShape.isRequired,
-    dashboardId: PropTypes.string,
-  };
-
-  static defaultProps = {
-    dashboardId: undefined,
+    activeProject: PropTypes.string.isRequired,
   };
 
   state = {
     isFullscreen: false,
+    dashboardName: '',
   };
 
   onChangeFullscreen = (isFullscreen) => {
     this.setState({ isFullscreen });
   };
 
+  onDashboardDataReceived = ({ name }) => {
+    this.setState({
+      dashboardName: name,
+    });
+  };
+
   toggleFullscreen = () => {
     this.setState({ isFullscreen: !this.state.isFullscreen });
   };
 
   render() {
-    const { formatMessage } = this.props.intl;
+    const {
+      activeProject,
+      intl: { formatMessage },
+    } = this.props;
+
+    const breadcrumbs = [
```

Comment:
I think i will be better to represent breadcrumb config like array of objects:
`
[
  {
    title: formatMessage(messages.pageTitle),
    to: { type: PROJECT_DASHBOARD_PAGE, payload: { projectId: activeProject } },
  },
  {
    title: formatMessage(messages.subPageTitle),
  },
]
`
And then check object property 'to'. If it's exists - draw link, no - draw plain text.

Reply:
makes sense. Will do